ios/BetterSocial/Info.plist

```diff
@@ -127,9 +125,5 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
-	<key>FirebaseDynamicLinksCustomDomains</key>
-	<array>
-		<string>https://link.bettersocial.org</string>
-	</array>
 </dict>
-</plist>
\ No newline at end of file
+</plist>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [128-130]

It seems like the `FirebaseDynamicLinksCustomDomains` key was removed from this part of the plist file. As long as it's defined elsewhere in the plist file (which it appears to be according to the new hunk from lines 32-64), this should be fine. However, please ensure that this doesn't affect the functionality of Firebase Dynamic Links in your app.


<!-- This is an auto-generated comment by OSS CodeRabbit -->